app/websocket/vault_socket.py

```
@@ -154,25 +154,92 @@ async def websocket_user_endpoint(
 async def websocket_vault_endpoint(
     websocket: WebSocket,
     hardware_uuid: str = Query(..., description="Vault hardware UUID"),
-    vault_secret: str = Query(..., description="Pre-shared vault secret"),
     repo: VaultRepository = Depends(get_vault_repo),
```

Ruff (0.15.2)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)